sys/dev/isp/isp.c

```diff
@@ -4662,15 +4858,468 @@ isp_parse_flt_2xxx(ispsoftc_t *isp, uint8_t *flt_data)
 	return (0);
 }
 
+static void
+isp_print_image(ispsoftc_t *isp, char *name, struct isp_image_status *image_status)
+{
+	isp_prt(isp, ISP_LOGDEBUG0,
+	    "%s %s: mask=0x%02x gen=0x%04x ver=%u.%u map=0x%01x sum=0x%08x sig=0x%08x",
+	    name, "status",
+	    image_status->image_status_mask,
+	    le16toh(image_status->generation),
+	    image_status->ver_major,
+	    image_status->ver_minor,
+	    image_status->bitmap,
+	    le32toh(image_status->checksum),
+	    le32toh(image_status->signature));
+}
+
+static bool
+isp_check_aux_image_status_signature(struct isp_image_status *image_status)
+{
+	unsigned long signature = le32toh(image_status->signature);
+
+	return (signature != ISP28XX_AUX_IMG_STATUS_SIGN);
+}
+
+static bool
+isp_check_image_status_signature(struct isp_image_status *image_status)
+{
+	unsigned long signature = le32toh(image_status->signature);
+
+	return
+	    signature != ISP27XX_IMG_STATUS_SIGN &&
+	    signature != ISP28XX_IMG_STATUS_SIGN;
```

Comment:
Why no parentheses?

Reply:
Sorry, missed that. Is
```
return
    ((signature != ISP27XX_IMG_STATUS_SIGN) &&
    (signature != ISP28XX_IMG_STATUS_SIGN));
```
better?

Reply:
I would remove the newline after "return".

Reply:
Ok